sheets/src/test/java/com/novoda/github/reports/sheets/convert/ContentHeaderRemoverTest.java

```diff
@@ -0,0 +1,41 @@
+package com.novoda.github.reports.sheets.convert;
+
+import com.novoda.github.reports.sheets.sheet.Entry;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ContentHeaderRemoverTest {
+
+    private static final String HEADER = "content:";
+
+    private final ContentHeaderRemover contentHeaderRemover = new ContentHeaderRemover();
+
+    @Test
+    public void givenAnEntryHavingContentWithHeader_whenRemoving_thenItIsRemoved() throws Exception {
+        Entry entry = new Entry("title", HEADER + " text");
+
+        Entry cleanEntry = contentHeaderRemover.removeFrom(entry);
+
+        assertThat(cleanEntry.getContent()).isEqualTo("text");
+    }
+
+    @Test
+    public void givenAnEntryNotHavingHeader_whenRemoving_thenNothingIsRemoved() throws Exception {
+        Entry entry = new Entry("title", "text");
+
+        Entry cleanEntry = contentHeaderRemover.removeFrom(entry);
+
+        assertThat(cleanEntry.getContent()).isEqualTo("text");
+    }
+
+    @Test
+    public void givenAnEntryHavingOnlyTheHeader_whenRemoving_thenEverythingIsRemoved() throws Exception {
+        Entry entry = new Entry("title", HEADER);
+
+        Entry cleanEntry = contentHeaderRemover.removeFrom(entry);
+
+        assertThat(cleanEntry.getContent()).isEqualTo("");
+    }
+}
```

Comment:
Because we don't do that for the title, but only for the content, as the name of the class says. However, we added one more test (maybe irrelevant to your question) to showcase what happens when there is more than one colon in the content: aab3b1a